tests/wicked/wlan/sut/t04_wpa_eap_peap_mschapv2.pm

```diff
@@ -55,23 +55,52 @@ has hostapd_conf => q(
     auth_server_shared_secret=testing123
 );
 
-has ifcfg_wlan => q(
-    BOOTPROTO='dhcp'
-    STARTMODE='auto'
+has ifcfg_wlan => sub { [
+        q(
+        BOOTPROTO='dhcp'
+        STARTMODE='auto'
 
-    # Global settings
-    WIRELESS_AP_SCANMODE='1'
-    WIRELESS_WPA_DRIVER='nl80211'
+        # Global settings
+        WIRELESS_AP_SCANMODE='1'
+        WIRELESS_WPA_DRIVER='nl80211'
 
-    # Network settings
-    WIRELESS_ESSID='{{ssid}}'
-    WIRELESS_AUTH_MODE='eap'
-    WIRELESS_EAP_AUTH='mschapv2'
-    WIRELESS_EAP_MODE='PEAP'
-    WIRELESS_MODE='Managed'
-    WIRELESS_WPA_ANONID='anonymous'
-    WIRELESS_WPA_IDENTITY='{{eap_user}}'
-    WIRELESS_WPA_PASSWORD='{{eap_password}}'
-);
+        # Network settings
+        WIRELESS_ESSID='{{ssid}}'
+        WIRELESS_AUTH_MODE='eap'
+        WIRELESS_EAP_AUTH='mschapv2'
+        WIRELESS_EAP_MODE='PEAP'
+        WIRELESS_MODE='Managed'
+        WIRELESS_WPA_ANONID='anonymous'
+        WIRELESS_WPA_IDENTITY='{{eap_user}}'
+        WIRELESS_WPA_PASSWORD='{{eap_password}}'
+    ),
+        q(
+        BOOTPROTO='dhcp'
+        STARTMODE='auto'
+
+        # Network settings
+        WIRELESS_ESSID='{{ssid}}'
+        WIRELESS_AUTH_MODE='eap'
+        WIRELESS_EAP_AUTH='mschapv2'
+        WIRELESS_EAP_MODE='PEAP'
+        WIRELESS_WPA_IDENTITY='{{eap_user}}'
+        WIRELESS_WPA_PASSWORD='{{eap_password}}'
+        WIRELESS_CA_CERT='{{ca_cert}}'
+    ),
+        {
+            wicked_version => '>=0.6.66',
```

Comment:
this kinda confused me here. i noticed that this file has a similar structure as the others. But for instance tests/wicked/wlan/sut/t05_wpa_eap_ttls.pm defined the variable `wicked_version` alone but here is as member of the `ifcfg_wlan`. 
shouldnt this assigned as the others?....

Reply:
You can specify the `wicked_version` in test-module context and only for a specific configuration.
If it is specified as member, it apply to the full test module, here it apply only for this specific config.